src/pages/phones-page/PhonesPage.tsx

```diff
@@ -9,8 +9,8 @@ const product: Product = {
   phoneId: 'string',
   itemId: 'string',
   name: 'string',
-  fullPrice: 1,
-  price: 1,
+  fullPrice: 799,
+  price: 799,
```

Comment:
Let's make this dynamic. we've introduced the api method under utils and services, check them out!